src/Dock.Avalonia/DockControlState.cs

```diff
@@ -66,7 +66,7 @@ private void Drop(Point point, DragAction dragAction, IVisual relativeTo)
 
         private void Leave()
         {
-            if (_dropControl is DockPanel)
+            if (_dropControl is Panel)
```

Comment:
We probably need to add custom Panel type here or something otherwise dock target is displayed on tabitem header